UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/ObstacleMenu.cs

```diff
@@ -0,0 +1,46 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+public class ObstacleMenu : IMenu
+{
+    private GUIContent obstaclePickerButtonTitle = new GUIContent("View Obstacles");
+
+    public void Display()
+    {
+        DisplayTitle();
+        DisplayObstaclesPickerButton();
```

Comment:
aren't we listening for the picker event? or is it not part of the deal now just visualising obstacle? :thinking:

Reply:
👍

Reply:
I WIN THE INTERNET